.github/workflows/oblt-github-commands.yml

```diff
@@ -0,0 +1,63 @@
+---
+##
+## This the automation to let Observability team members to know what are the
+## supported GitHub commands to interact with the Observability test environments.
+##
+## Owner: @elastic/observablt-robots
+##
+name: oblt-github-commands
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+permissions:
+  contents: read
+
+jobs:
+  comment-if-apm-ui:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: elastic/apm-pipeline-library/.github/actions/github-token@current
+        with:
+          url: ${{ secrets.OBLT_VAULT_ADDR }}
+          roleId: ${{ secrets.OBLT_VAULT_ROLE_ID }}
+          secretId: ${{ secrets.OBLT_VAULT_SECRET_ID }}
+
+      - id: is_team_member
+        name: Check if user is member of the Elastic org and apm-ui team
+        run: |
+          if gh api -H "Accept: application/vnd.github+json" \
+            /orgs/elastic/teams/apm-ui/memberships/${{ github.actor }} ; then
```

Comment:
there is also \@observability-ui

Reply:
> there is also @observability-ui

Cool, thanks Dario, both teams are under the meta-group `Observability`, so this automation is now using the meta-team instead of `apm-ui` only.